src/cascadia/TerminalApp/CommandPalette.idl

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import "../Command.idl";
+
+namespace TerminalApp
+{
+    [default_interface] runtimeclass CommandPalette : Windows.UI.Xaml.Controls.Grid
+    {
+        CommandPalette();
+
+        Windows.Foundation.Collections.IObservableVector<Command> FilteredActions { get; };
+
+        void SetActions(Windows.Foundation.Collections.IVector<Command> actions);
+
+        void ToggleVisibility();
+        void SetDispatch(ShortcutActionDispatch dispatch);
```

Comment:
I was going to ask if we should ctor with the dispatch, but we can't because xaml